rest/model/distro.go

```diff
@@ -354,6 +354,7 @@ type APIDistro struct {
 	DisableShallowClone   bool                     `json:"disable_shallow_clone"`
 	UseLegacyAgent        bool                     `json:"use_legacy_agent"`
 	Note                  APIString                `json:"note"`
+	ValidProjects         []string                 `json:"valid_projects"`
```

Comment:
I don't know the pros and cons, but there's a function ToAPIStringList (and From) if you want to do that

Reply:
ah, didn't realize, fixed